### ISSUE TITLE
Fix: Add `aze-cli reset` account to clear out current account fields (As this would really be handy while testing stuff)

### ISSUE THREAD

**Issue by nlok5923** [RizeLabs/aze-cli#16]

**Comment by nlok5923** [comment]
@Created-for-a-purpose maybe you could prioritize implementin this first for branch: `v-plaintext`

**Comment by nlok5923** [comment]
Also, make sure to remove the `Player.toml and ws_config.json` just mentioning

**Comment by Created-for-a-purpose** [comment]
I don't think entire Player.toml needs to be removed, just the game_id field.  Also, ig there are only 2 slots in player account that have data, that too will just change on aze-cli init, makes me rethink if reset is really needed?

**Comment by nlok5923** [comment]
Yeah for plaintext version i think reset might not make much sense. But for the encryption one it'll

**Comment by nlok5923** [comment]
@Created-for-a-purpose i think we should work on this, Like along with game field it should also delete the client store (for now) as that would be quite handy for testing.
